frontend_app_file/test/fixture/content/content.js

```diff
@@ -0,0 +1,12 @@
+import { file } from './file.js'
+import { commentList, revisionList } from 'tracim_frontend_lib/dist/tracim_frontend_lib.testUtils.js'
+import { shareList } from './shareList.js'
+
+const content = {
```

Comment:
`content` is too generic, and would fit best only for the `file` variable of this file.
Try to give it a more precise name